content/docs/guides/responsive_amp/responsive_design@zh_CN.md

```diff
@@ -0,0 +1,173 @@
+---
+$title: 制作自适应 AMP 网页
+---
+[TOC]
+
+## 简介
+
+自适应设计就是制作可根据用户需求自行调整的灵活网页（即：能够适应用户设备的屏幕尺寸和方向的网页）。通过 AMP，您可以轻松实现这一目的。AMP 支持所有的屏幕类别和设备类别，并会提供内置的自适应组件。
+
+在本指南中，我们会向您展示如何在 AMP 中轻松实现下列自适应基本功能：
+
+- [控制视口](#控制视口)
+- [创建自适应布局](#创建自适应布局)
+- [缩放媒体](#为网页缩放媒体)
+
+## 控制视口
+
+要想优化网页以便内容能够自动缩放并适应任何设备的浏览器窗口，您需要指定 `meta` 视口元素。视口元素会指示浏览器如何缩放网页的可见区域（视口）以及如何调整其大小。
+
+不过，应该使用哪些值呢？对于这个问题，AMP 中已给出明确答案。您需要指定以下视口作为 AMP 网页[必需标记](/zh_cn/docs/reference/spec.html#required-markup)的一部分：
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+以上是您在制作自适应网站时需指定的一般视口设置。虽然 `initial-scale=1` 并不是有效的 AMP 网页所必需的设置，我们仍想建议您指定此项，因为它会在网页首次加载时将缩放级别设为 1。
+
+## 创建自适应布局
+
+在自适应设计中，您可以使用 CSS [`@media`](https://developer.mozilla.org/docs/Web/CSS/@media) 查询来针对各种屏幕尺寸定制网页样式，而无需更改网页内容。在 AMP 中，您可以继续使用相同的 CSS `@media` 查询。此外，若想更精细地控制某个 AMP 元素，您可为该元素指定 `media` 属性。如果您需要根据媒体查询显示或隐藏某个元素，这项设置会特别有用。请参阅[更改图片的艺术设计](#changing-the-art-direction-of-an-image)部分，查看使用 `media` 属性的示例。
```

Comment:
Translate `#changing-the-art-direction-of-an-image` -> Should correspond to header within this doc.